core/domain/platform_parameter_list.py

```diff
@@ -70,6 +70,7 @@ class ParamNames(enum.Enum):
     platform_parameter_domain.FeatureStages.PROD)
 
 Registry.create_feature_flag(
-    ParamNames.BLOG_PROJECT,
-    'This flag is for blog project pages.',
+    ParamNames.BLOG_PAGES,
+    'This flag is for blog home page, blog author profile page and blog post' +
+    'page.',
```

Comment:
Missing space between "post" and "page".

Reply:
Done.